src/neo/Cryptography/ECC/ECPoint.cs

```diff
@@ -22,15 +22,16 @@ public bool IsInfinity
 
         internal ECPoint(ECFieldElement x, ECFieldElement y, ECCurve curve)
         {
-            if ((x != null && y == null) || (x == null && y != null))
-                throw new ArgumentException("Exactly one of the field elements is null");
+            if ((x != null && y is null) || (x is null && y != null) || (x is null && y is null && curve is null))
+                throw new ArgumentException("The field element is null");
```

Comment:
Why change this line?

Reply:
in case of all the three elements are null. for the origin `Exactly one of the field elements is null`, it seems to mean only one element is null.